### DISCUSSION TITLE
[@types/chart.js] is the moment dependency necessary?

### DISCUSSION THREAD

**Discussion by kitsunekyo** [DefinitelyTyped/DefinitelyTyped#64427] [Issues with a @types package]
is the dependency on `moment` really necessary for the types package? shouldnt the adapters alone bring in more specific types that can be used?

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/chart.js/package.json#L4

[a comment by an automated account is not shown]

**Reply by lekoaf** [reply]
Maybe. You can try it.
